.github/workflows/publish_image.yml

```diff
@@ -1,11 +1,9 @@
 name: Publish to Image Registry
 
 on:
-  push:
-    branches:
-      - 'studio'
-    tags:
```

Comment:
Do we no longer create tags? Should we still run it on tag creation?

Reply:
Yes I think we should keep these so that our [release announcements](https://github.com/supabase/supabase/releases) are accompanied by docker hub image tag.

Reply:
Fixed.